xbmc/addons/AddonManager.cpp

```diff
@@ -960,16 +960,26 @@ bool CAddonMgr::CanAddonBeInstalled(const AddonPtr& addon)
 
 bool CAddonMgr::CanUninstall(const AddonPtr& addon)
 {
-  return addon && !IsSystemAddon(addon->ID()) && CanAddonBeDisabled(addon->ID()) &&
-         !StringUtils::StartsWith(addon->Path(),
-                                  CSpecialProtocol::TranslatePath("special://xbmc/addons"));
+  return addon && CanAddonBeDisabled(addon->ID()) && !IsBundledAddon(addon->ID());
+}
+
+bool CAddonMgr::IsBundledAddon(const std::string& id)
+{
+  return XFILE::CDirectory::Exists(
+             CSpecialProtocol::TranslatePath("special://xbmc/addons/" + id + "/")) ||
+         XFILE::CDirectory::Exists(
+             CSpecialProtocol::TranslatePath("special://xbmcbin/addons/" + id + "/"));
```

Comment:
for my particular setup those two paths are the same, there may be installations in the wild where they differ. i think an OR'ed check is fine here.